Client/Frontend/Settings/SearchSettingsTableViewController.swift

```diff
@@ -104,8 +104,9 @@ class SearchSettingsTableViewController: ThemedTableViewController {
             case ItemDefaultSuggestions:
                 cell.textLabel?.text = .SearchSettingsShowSearchSuggestions
                 cell.textLabel?.numberOfLines = 0
-                let toggle = UISwitch()
-                toggle.onTintColor = themeManager.currentTheme.colors.actionPrimary
+                let toggle = ThemedSwitch()
+                toggle.applyTheme(theme: themeManager.currentTheme)
+                toggle.isEnabled = true
```

Comment:
I'm not sure those switched should be changed to be enabled to true by default? What's the reasoning behind this change?

Reply:
So let me better explain my comment 😅 That value is [true by default](https://developer.apple.com/documentation/uikit/uicontrol/1618217-isenabled). So I don't think we need to set it to true here agian.

Reply:
Got it 😇! 
Will make the changes.